Tests/Test.cs

```diff
@@ -0,0 +1,191 @@
+﻿using Mars_Luiz.Pages;
+using Mars_Luiz.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.DevTools.V123.Storage;
+using OpenQA.Selenium.Support.UI;
+
+namespace Mars_Luiz.Tests
+
+{
+    [TestFixture]
+    internal class Mars : CommonDriver
+    {
+        [SetUp]
+        public void SignSignInSteps()
```

Comment:
you have implemented the test in wrong framework, it should be done in specflow not in NUnit

Reply:
this is not needed you can remove thi s